Makefile

```
@@ -1,66 +1,57 @@
-ESPHOME_PATH = ../esphome
-ESPHOME_REF = 2025.7.5
-PAGEFIND_VERSION=1.1.1
-PAGEFIND=pagefind
-NET_PAGEFIND=../pagefindbin/pagefind
+.PHONY: clean live-html check-links anchors production convert-from-rst convert-branch-in-place directories netlify repo-data all
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)